test/ResourcesClassLibraryNoAttribute/project.json

```diff
@@ -0,0 +1,17 @@
+﻿{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk"
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
Because if I do I get a warning that it ended up with NETStandard.Library 1.6.0 anyway. What are the downside of using 1.6 considering that this a a test library?